components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -89,8 +89,8 @@ const ContributorHighlightCard = ({ title, desc, prLink, user, id }: Contributor
         );
         setLoading(false);
         if (res) {
-          ToastTrigger({ message: "Highlights Updated Successfully", type: "success" });          
-          mutate(`users/${user}/highlights`);          
+          ToastTrigger({ message: "Highlights Updated Successfully", type: "success" });
+          mutate(`users/${user}/highlights`);
           setOpen(false);
           setTimeout(() => {
             document.body.setAttribute("style", "pointer-events:auto !important");
```

Comment:
IMHO this is not part of this release but should be looked at maybe in a tooling issue, programatic styles are "meh" at best; even if we have to use the same method to apply the properties, we should work around tailwind classes rather than setting manual styles, applying the same pattern in multiple ui blocking components will result in chaos 😅